src/main/java/com/openlattice/edm/type/PropertyType.java

```diff
@@ -53,7 +55,8 @@ public PropertyType(
             @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype,
             @JsonProperty( SerializationConstants.PII_FIELD ) Optional<Boolean> piiField,
             @JsonProperty( SerializationConstants.MULTI_VALUED ) Optional<Boolean> multiValued,
-            @JsonProperty( SerializationConstants.ANALYZER ) Optional<Analyzer> analyzer ) {
+            @JsonProperty( SerializationConstants.ANALYZER ) Optional<Analyzer> analyzer,
+            @JsonProperty( SerializationConstants.INDEXED ) Optional<Boolean> postgresIndexed ) {
```

Comment:
My previous comments seem to have disappeared. This should be an enum with NONE, BTREE, and GIN as the possible options.

Reply:
(think that comment was on the conductor-client branch) --  sounds good!